scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,190 @@
+        @content;
+      }
+    }
+  } @else {
+    @if $divider-order == 1 {
+      &.p-equal-height-row--has-1st-divider .p-equal-height-row__col::before {
+        @content;
+      }
+    }
+
+    @if $divider-order == 2 {
+      &.p-equal-height-row--has-2nd-divider .p-equal-height-row__col::after {
+        @content;
+      }
+    }
+
+    @if $divider-order == 3 {
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider) .p-equal-height-row__col::before,
+      &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider) .p-equal-height-row__col::after {
+        @content;
+      }
+    }
+  }
+}
+
+@mixin divider-styles($large-screen: false) {
+  // For each row or column grid we only have access to two pseudo elements
+  // if 1st-divider (::before) is present, assume 2nd-divider (::after) isn't, then 3rd-divider must be (::after)
+  // if 2nd-divider (::after) is present, assume 1st-divider (::before) isn't, then 3rd-divider must be (::before)
+  @if $large-screen {
+    &.p-equal-height-row--has-1st-divider::before,
+    &.p-equal-height-row--has-2nd-divider::after,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider)::before,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider)::after {
+      @content;
+    }
+  } @else {
+    // For smaller screens, the divider pseudo elements are inserted at the column level
+    &.p-equal-height-row--has-1st-divider .p-equal-height-row__col::before,
+    &.p-equal-height-row--has-2nd-divider .p-equal-height-row__col::after,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-1st-divider) .p-equal-height-row__col::before,
+    &.p-equal-height-row--has-3rd-divider:not(.p-equal-height-row--has-2nd-divider) .p-equal-height-row__col::after {
+      @content;
+    }
+  }
+}
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
+    position: relative;
+
+    .p-equal-height-row__col {
+      // smaller screens each column will have border top by default
+      border-top-color: $colors--theme--border-default;
+      border-top-style: solid;
+      border-top-width: 1px;
+      display: grid;
+      grid-column: span $grid-columns-small;
+      grid-row: span 4;
+      grid-template-rows: subgrid;
+      margin-top: -1px;
+      position: relative;
+
+      @media screen and ($breakpoint-small <=width < $breakpoint-large) {
+        grid-column: span $grid-columns-medium;
+        grid-template-columns: subgrid;
+
+        // for medium screen, each column item will take half of the available cols from the parent grid
+        .p-equal-height-row__item {
+          grid-column: span calc($grid-columns-medium / 2);
+        }
+
+        // for medium screen, position the first column item on the left of the grid
+        .p-equal-height-row__item:first-child {
+          grid-row: span 100;
```

Comment:
I guess it doesn't have to be 100? Let's make it just that it needs to be. So 3 I guess (if we have 4 items max in total, then we can have only 3 in the split view).

Reply:
nice catch, this was set to 100 before we had the discussion to have maximum of 4 items per column. 3 is perfect